src/components/user-research-banner/user-research-banner.browser.test.js

```diff
@@ -1,25 +1,14 @@
-/**
- * @jest-environment ./lib/puppeteer/environment.js
- */
-/* eslint-env jest */
-
 const configPaths = require('../../../config/paths.json');
 
 describe('/components/user-research-banner', () => {
-  let page;
-  const url = `http://localhost:${configPaths.ports.test}/components/user-research-banner/default/preview`;
+  const url = `http://localhost:${configPaths.ports.app}/components/user-research-banner/default/preview`;
```

Comment:
I didn't think we needed a separate test port - we can just reuse the app one - and the jest can skip spinning up it's own dev server if ones already running